kotlin/workflow-core/src/main/java/com/squareup/workflow/Worker.kt

```diff
@@ -381,3 +393,16 @@ internal class TypedWorker<T>(
 
   override fun toString(): String = "TypedWorker(key=\"$key\", type=$type)"
 }
+
+private class TimerWorker(
+  private val delayMs: Long,
```

Comment:
On iOS, the convention is to use floating-point number of seconds, which our implementation should follow. Probably fine to have differences on this, but wanted to note it in case others feel differently.

Reply:
Agree that this is a reasonable difference. A `Long` of ms is definitely idiomatic for java / kotlin.